modules/ingest-geoip/src/internalClusterTest/java/org/elasticsearch/ingest/geoip/GeoIpDownloaderIT.java

```diff
@@ -463,6 +512,9 @@ private void putGeoIpPipeline(String pipelineId) throws IOException {
                             builder.field("field", "ip");
                             builder.field("target_field", "ip-city");
                             builder.field("database_file", "GeoLite2-City.mmdb");
+                            if (downloadDatabaseOnPipelineCreation) {
+                                builder.field("download_database_on_pipeline_creation", true);
```

Comment:
I think you want the ability to set this to false for the test right? It defaults to true.

Reply:
Good catch.
The condition has been updated to:

```
if (downloadDatabaseOnPipelineCreation == false || randomBoolean()) {
```

The random boolean allow to randomize testing between `download_database_on_pipeline_creation ` missing or set to `true`